control_panel_api/models.py

```diff
@@ -112,3 +112,9 @@ class AppS3Bucket(TimeStampedModel):
     class Meta:
         # one record per app/s3bucket
         unique_together = ('app', 's3bucket')
+
+    def has_readwrite_access(self):
+        return self.access_level == self.READWRITE
+
+    def update_aws_permissions(self):
+        services.apps3bucket_update(self)
```

Comment:
So here, this ties with the convo we had with Andy other day about the model coupling in service library.  The goal of my refactoring is to remove the model from the services and just let the service functions accept parameters (could then be used elsewhere in commands and reduces coupling).  So this method should not pass `self`, but pass its attributes as parameters to `apps3bucket_update`.  That method would require a slight rewrite to accept params.

And the function name now change from `apps3bucket_update ` to something like `aws_update`, but we still need to decide on them.